packages/amplify-gen1-codegen-function-adapter/API.md

```diff
@@ -0,0 +1,39 @@
+## API Report File for "@aws-amplify/amplify-gen1-codegen-auth-adapter"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AuthDefinition } from '@aws-amplify/amplify-gen2-codegen';
+import { LambdaConfigType } from '@aws-sdk/client-cognito-identity-provider';
+import { PasswordPolicyPath } from '@aws-amplify/amplify-gen2-codegen';
+import { UserPoolType } from '@aws-sdk/client-cognito-identity-provider';
```

Comment:
are we using cognito types in functions ?

Reply:
No, updated the API.md file